lib/afc/streams.js

```diff
@@ -20,21 +20,23 @@ class AfcReadableFileStream extends stream.Readable {
     }
 
     if (_.isEmpty(data)) {
-      await this._destroy();
+      this.destroy();
```

Comment:
shouldn't it be awaited?

Reply:
Not this method. This the public api from Stream